languages/example/nickserv.lang.json

```diff
@@ -20,7 +20,7 @@
   "$bUNREGISTER$b lets you delete your user account.": "$bUNREGISTER$b lets you delete your user account.",
   "$bVERIFY$b lets you complete account registration.": "$bVERIFY$b lets you complete account registration.",
   "Invalid parameters": "Invalid parameters",
-  "NickServ lets you register and login to an account.\n\nTo see in-depth help for a specific NickServ command, try:\n    $b/NS HELP <command>$b\n\nHere are the commands you can use:\n%s": "NickServ lets you register and login to an account.\n\nTo see in-depth help for a specific NickServ command, try:\n    $b/NS HELP <command>$b\n\nHere are the commands you can use:\n%s",
+  "NickServ lets you register and log into an account.": "NickServ lets you register and log into an account.",
```

Comment:
we should make this "log in to", reads a bit more clearly imo ([ref](https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to))